docs/en/observability/monitor-k8s/monitor-k8s.asciidoc

```diff
@@ -0,0 +1,74 @@
+[[monitor-kubernetes]]
+= Monitor Kubernetes: Observe the health and performance of your Kubernetes deployments
+++++
+<titleabbrev>Monitor Kubernetes</titleabbrev>
+++++
+
+Applications running in a containerized environment like Kubernetes pose a
+unique monitoring challenge: how do you diagnose and resolve issues with
+hundreds of microservices on thousands (or millions) of containers, running
+in ephemeral and disposable pods?
+
+A successful Kubernetes monitoring solution has a few requirements:
+
+* Monitors all layers of your technology stack, including:
+** The host systems where Kubernetes is running.
+** Kubernetes core components, nodes, pods, and containers running within
+the cluster.
+** All the applications and services running in Kubernetes containers.
+
+* Automatically detects and monitors services as they appear dynamically.
+
+* Provides a way to correlate related data so that you can group and explore
+related metrics, logs, and other observability data.
+
+
+[discrete]
+== What you’ll learn
+
+This guide describes how to use Elastic {observability} to observe all layers of
+your application, including the orchestration software itself:
+
+* Collect logs and metrics from Kubernetes and your applications
+* Collect trace data from applications deployed with Kubernetes
+* Centralize the data in the {stack}
+* Explore the data in real-time using tailored dashboards and {observability} UIs
+
+//TODO: Add APM goals here
```

Comment:
```suggestion
```

Reply:
I figured I'd remove these in my follow-up PR, but might as well get rid of 'em now